docs/docs/metrics.md

```diff
@@ -258,3 +258,126 @@ Dashes in metric names are replaced with underscores.
   state.
 * `marathon.threads.terminated.gauge` —
   the number of threads in `TERMINATED` state.
+
+## Alerting
+
+Let's consider a few examples of alerting rules. We will use Prometheus
+as our alerting tool, but please keep in mind that any alerting service
+of your choice can be used instead.
+
+Please note that particular threshold values are dependent on many
+variables like the number of applications, a cluster size and so on.
+
+For the syntax of Prometheus alerting rules please refer to
+[its official documentation](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
+
+Alert whenever a Marathon instance goes down or restarts:
+
+```yaml
+  - alert: Marathon Uptime
+    expr: up{job="marathon"} == 0
+    for: 30s
+    labels:
+      severity: warning
+    annotations:
+      summary: It fires when a Marathon instance stops or restarts.
+```
+
+Alert when there are less than 2 out of 3 Marathon instances that are
+up and running:
+
+```yaml
+  - alert: Marathon Too Few Instances
+    expr: sum(up{job="marathon"}) < 2
+    for: 30s
+    labels:
+      severity: critical
+      annotations:
+      summary: It fires when there are fewer Marathon instances up and running than expected.
+```
+
+Alert if Marathon's heap usage is too high:
```

Comment:
add `... is too high ( >5Gb)`

Reply:
These high thresholds are based on the metric values from our performance tests. Do you think we should target the use-case when there is only a handful of apps?

Reply:
I think this is a meaningful value. Afaik we recommend 2Gb and I've almost never seen Marathon exceeding 5Gb in "normal circumstances" so the values sgtm.